vdc/k8s-config.yaml

```diff
@@ -189,6 +189,18 @@ metadata:
   name: test-svc
   namespace: batch-pods
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: deploy-svc
```

Comment:
I think this is meant to be test-svc

Reply:
Never mind, don’t need to be unique across namespaces https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/

Reply:
Will approve without this: What do you think about using one service account here, and just issuing RoleBindings for the needed resources in the test namespace?